Add sensu logging to /var/log/sensu/ on systemd based systems. (#1355)
sensu/sensu-docs: content/sensu-go/5.5/guides/troubleshooting.md

Sensu services print [structured log messages][structured] to standard output.
In order to capture these log messages to disk or another logging facility, Sensu services
make use of capabilities provided by the underlying operating system's service
management. For example, logs are sent to the journald when systemd is the service manager,
whereas log messages are redirected to `/var/log/sensu` when running under sysv

> init schemes. If you are running systemd as your service manager and would rather have logs written to `/var/log/sensu/`, you can follow our guide on forwarding logs from [journald to syslog][journald-syslog].

Reviewer: ```suggestion
init schemes. If you are running systemd as your service manager and would rather have logs written to `/var/log/sensu/`, see the guide to [forwarding logs from journald to syslog][journald-syslog].
```
